src/main/java/bio/terra/service/common/gcs/BigQueryUtils.java

```diff
@@ -0,0 +1,15 @@
+package bio.terra.service.common.gcs;
+
+import bio.terra.service.snapshot.Snapshot;
+import bio.terra.stairway.FlightContext;
+
+public class BigQueryUtils {
+
+  public static String getSuffix(FlightContext context) {
+    return context.getFlightId().replace('-', '_');
+  }
+
+  public static String firestoreDumpTableName(Snapshot snapshot) {
```

Comment:
We've been calling this a staging table, so maybe something like "gspathStagingTableName"